src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/IdentityTexlVisitor.cs

```diff
@@ -5,77 +5,97 @@
 
 namespace Microsoft.PowerFx.Core.Syntax.Visitors
 {
-    // Identity visitor
-    internal abstract class IdentityTexlVisitor : TexlVisitor
+    /// <summary>
+    /// Visitor that by default does nothing. <seealso cref="TexlVisitor" />.
+    /// </summary>
+    public abstract class IdentityTexlVisitor : TexlVisitor
```

Comment:
Did not make this public earlier, but it seems it is useful, and it doesn't expose any new things really.